web_widget_timepicker/__openerp__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this be changed to the standard __manifest__.py?

Reply:
Have you review the older version it not v14.